ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -187,45 +187,46 @@ define([
             };
         },
 
-        showTabs: function () {
+        showTabs: function (root) {
             // Set up tab events, show only first child
-            $('.tabs a').each(function (el, i) {
-                var tabGroup = el.getAttribute('href');
-                var obj;
-                var AdStatus;
+            var tabContainer = $('.tabs');
+            var tabs = $('a',tabContainer);
+
+            tabs.each(function(tab,i) {
                 if (i > 0) {
-                    $(tabGroup).hide();
+                    $(tab.getAttribute('href')).hide();    
                 }
-                bean.on(el, 'click', function (e) {
-                    e.preventDefault();
+            });
+
+            bean.on(tabContainer[0], 'click', 'a', function (e) {
+
+                e.preventDefault();                
+                var tab = $(this);
 
-                    $($('[aria-selected="true"]').attr('href')).hide();
-                    $('[aria-selected="true"]').attr("aria-selected", false);
+                if( tab.attr("aria-selected") !== 'true' ) {
+                 
+                    var activeTab = $('[aria-selected="true"]', tabContainer);
+                    $(activeTab.attr('href')).hide();
+                    activeTab.attr("aria-selected", false);
 
-                    $($(this).attr('href')).show();
-                    $(this).attr("aria-selected", true);
+                    $(tab.attr('href')).show();
```

Comment:
It's because the `href` is in the form `#someid`, so if you do `$(elem.attr('href'))` is like doing `$('#someid')`. It's maybe not so elegant, I agree.